components/collapsible/composed/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+class CollapsibleComposed extends Component {
+  state = { collapsed: this.props.collapsed }
```

Comment:
Does it really work ????
In this case `this` is window...

Reply:
In that case, `this` is the Component itself, so it's correct.

Reply:
This is working, but I'll move it to a constructor so that it's safer.

Reply:
I wasn't updated ^^